dff/messengers/telegram/interface.py

```diff
@@ -7,7 +7,7 @@
 import asyncio
 from typing import Any, Optional, List, Tuple, Callable
 
-from telebot import types, apihelper
+from dff.messengers.common.modules import telegram
```

Comment:
Is it possible to import them via `from dff.messengers.common.modules.telegram import types, apihelper`.
That would result in way less changed lines.

Unless you want imports to look that way which is fine by me.

Reply:
No, it is not possible, since ` dff.messengers.common.modules` is not a package. At least otherwise did not work for me.